src/features/BlockReturner/index.js

```diff
@@ -4,6 +4,7 @@ import FeaturedBenefits from "Components/FeaturedBenefits";
 import FeaturedServices from "Components/FeatServices";
 import MissionStatement from "Components/MissionStatement";
 import FAQs from "Components/FAQs";
+import VisualList from "Components/VisualList";
```

Comment:
Use the loadable components import here, you can see it in the development branch:
`import loadable from "@loadable/component";`
`const VisualList = loadable(() => import("Components/VisualList"));`